datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/__init__.py

```diff
@@ -9,8 +9,9 @@
 from .prometheus import prom
```

Comment:
Need to add `jpype1` dep?

Reply:
No, similar to https://github.com/DataDog/integrations-core/blob/54e695283daf6b305ffdcd9e11cbe15984b3cea1/datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp.py#L44-L49